src/alchemlyb/parsing/gmx.py

```diff
@@ -20,6 +20,8 @@ def extract_u_nk(xvg, T):
         Path to XVG file to extract data from.
     T : float
         Temperature in Kelvin the simulations sampled.
+    filter : bool
+        Filter out the lines that cannot be parsed.
```

Comment:
described the types of lines that are filtered

Reply:
also mention if `filter=True` is slower; add any information that users need to have to make a decision if to use `filter=True`

Reply:
Is it slower, and if so, by how much?